script8.js

```diff
@@ -0,0 +1,36 @@
+function change() {
+    window.location.href = 'https://www.google.com';
+}
+function goBack() {
+    window.history.back();
+}
+
+window.onload = function(){
+    document.getElementById("browser").innerHTML =  window.navigator.userAgent;
+    document.getElementById("version").innerHTML =  window.navigator.appVersion;
+    document.getElementById("height").innerHTML =  window.screen.height;
+    document.getElementById("width").innerHTML =  window.screen.width;
+    
+    document.getElementById('cookie').addEventListener('click',(e) =>{
+        let cookieName = "My Cookie";
+        let cookieValue = "Cookie Value";
+        document.cookie = cookieName + " = " + cookieValue;
```

Comment:
There should be no space after and before "=" symbol.

Reply:
Please note! If you have other cookies already set, this function will override them